go.mod

```diff
@@ -87,8 +87,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.11.1
 	istio.io/api v0.0.0-20230309115320-7befbde0f0dd
-	istio.io/client-go v1.17.2-0.20230309120219-62ca95f08159
-	istio.io/pkg v0.0.0-20230217221449-ee41b8dd085b
+	istio.io/client-go v1.17.1-0.20230217222049-183e0a2c5a1f
```

Comment:
This change is wrong, back-leveling the client-go.

Reply:
Had another run at this and looks like it is picking the right changes.

Reply:
Looks good now.